graphql2/schema.graphql

```diff
@@ -705,6 +705,12 @@ type AlertLogEntry {
   id: Int!
   timestamp: ISOTimestamp!
   message: String!
+  state: AlertLogEntryState
+}
+
+type AlertLogEntryState {
+  details: String!
+  status: String
```

Comment:
Instead of a string, maybe `'OK' | 'ERR'` as an enum. Will make it a lot easier for consumers of the API to know/understand what to expect